README.md

```diff
@@ -2,40 +2,71 @@
 
 With the Cephalopod package you'll be able to easily track and manage the various instances of your application, as well as analizing interesting statistical information.
 
-## Installation
+## Installation (production)
 
-To install this package, first of all head to your code directory and clone this repository:
+First of all, we recommend that you install Cephalopod in a virtual environment:
 
 ```sh
-git clone https://github.com/indico/cephalopod.git
-cd cephalopod
+$ virtualenv env
+$ . env/bin/activate
```

Comment:
I'd use `source` instead of `.` - at least in zsh you cannot use `.` with a relative path unless you start the path with `./` (or make it absolute)